playbooks/installation/ldap_integration.adoc

```diff
@@ -20,259 +84,166 @@ toc::[]
+[source,bash]
 ----
-openshift_master_identity_providers=[{'name': 'my_ldap_provider', 'challenge': 'true', 'login': 'true', 'kind': 'LDAPPasswordIdentityProvider', 'attributes': {'id': ['dn'], 'email': ['mail'], 'name': ['cn'], 'p\
-referredUsername': ['uid']}, 'bindDN': '', 'bindPassword': '', 'ca': '/path/to/my-ldap-cacert.crt', 'insecure': 'false', 'url': 'ldap://ldap.mycompany.com:389/ou=users,dc=mycompany,dc=com?uid'}]
+systemctl restart atomic-openshift-master-api
 ----
 
-== Adding a filter to a specific group
+IMPORTANT: These configurations must be applied on each master instance
+
+=== OpenShift Ansible Configuration
 
-More than likely the logins to Openshift will be limited by a group.  This may include limiting the capabilities to certain groups.  We will discuss Group Syncing Later.
+The Ansible based OpenShift installer contains functionality to automate the configuration of the `LDAPPasswordIdentityProvider`. Several link:https://docs.openshift.com/container-platform/latest/install_config/install/advanced_install.html#configuring-cluster-variables[cluster variables] of the Ansible link:https://docs.openshift.com/container-platform/latest/install_config/install/advanced_install.html#configuring-ansible[inventory] can be specified to tune the configuration of identity providers. For LDAP, these include:
 
-We will assume we are connection to a standard ldap.  The first step is to create a query to limit the user to a group, so from above we have:
+* `openshift_master_identity_providers` - An array of identity providers to configure within the OpenShift master configuration file
+* `openshift_master_ldap_ca` - Text value of the LDAP CA
+* `openshift_master_ldap_ca_file` - Location of the LDAP CA file on the Ansible control host. This file will be copied to the OpenShift master configuration directory (`/etc/origin/master`)
 
-[source,conf]
+The following can be added to the `[OSEv3:vars]` section of the OpenShift inventory file to configure the `LDAPPasswordIdentityProvider` for the LDAP structure referenced above. 
+
+[source,yaml]
 ----
-(&(objectClass=posixAccount)(memberof=CN=openshift,OU=groups,DC=example,DC=com))
+[OSEv3:vars]
+...
+openshift_master_identity_providers=[{'name': 'my_ldap_provider', 'challenge': 'true', 'login': 'true', 'kind': 'LDAPPasswordIdentityProvider', 'attributes': {'id': ['dn'], 'email': ['mail'], 'name': ['cn'], 'preferredUsername': ['uid']}, 'bindDN': '', 'bindPassword': '', 'ca': 'my-ldap-ca-bundle.crt', 'insecure': 'false', 'url': 'ldap://ldap.example.com:389/uid=users,dc=example,dc=com?uid'}]
+openshift_master_ldap_ca_file=/home/myuser/my-ldap-ca-bundle.crt
+...
 ----
 
-so in english this looks like:
+Run the `ansible-playbook` command referencing the inventory file and playbook which will be used to apply the changes to the cluster.
 
-Where object class equals posixAccount and the user is a member of the openshift group.
+[source,yaml]
+----
+ansible-playbook -i <inventory-file> <playbook>
```

Comment:
Can we be more specific here? I would like this guide to be a continuation of the install guide, so could align this with the fake `c1-ocp.myorg.com` cluster would be good.